paddle/fluid/distributed/auto_parallel/spmd_rules/elementwise_spmd_rule.cc

```diff
@@ -1,213 +0,0 @@
-/* Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
not rm, use git mv instead so that git maintain the update and commits from very beginning.

Reply:
git automatically modified it to "new" and "delete" when the files change a lot.